server/src/main/java/com/defold/extender/ExtenderUtil.java

```diff
@@ -26,14 +27,26 @@
 
 public class ExtenderUtil
 {
+    static Pattern createPattern(String expression) {
+        Pattern p = null;
+        try {
+            p = Pattern.compile(expression);
```

Comment:
For some cases, e.g. the pruning pattern is a good regex that we'll use later (e.g. "*.jar")

But in this case, I needed to also support a string like "{{env.SDK}}/path/to/lib.a", but these are reserved characters.

This issue popped up now with the latest refactoring, and this was a case I missed before.

Reply:
I am not sure this is the best solution, but it seems to work for now at least.